docs/apm/services.asciidoc

```diff
@@ -12,3 +12,32 @@ and requires anomaly detection to be enabled.
 
 [role="screenshot"]
 image::apm/images/apm-services-overview.png[Example view of services table the APM app in Kibana]
+
+[float]
+[[service-groups]]
+==== Service groups
+
+preview::[]
+
+Group services together to build meaningful views that remove noise and simplify investigations across services.
+Service groups are {kib} space-specific and available for any users with appropriate access.
+
+[role="screenshot"]
+image::apm/images/apm-service-group.png[Example view of service group in the APM app in Kibana]
+
+To enable Service groups, open {kib} and navigate to **Stack Management** > **Advanced Settings** > **Observability**,
+and enable the **Service groups feature**.
+
+To create a service group, navigate to **Observability** > **APM** > **Services** and select **Create group**.
+Specify a name, color, and description.
+Then, using the <<kuery-query, Kibana Query Language (KQL)>>, specify a query to select services for the group.
+Services that match the query within the last 24 hours will be assigned to the group.
```

Comment:
Maybe is worth mentioning that new services, even though they match the query used to create a group, won't show up automatically within that group.

Reply:
Ah! I didn't know that. Is the reverse true as well? Say a service matches at the time the group is created, but later doesn't, will it still show up in the group?

Reply:
it will not--a current limitation of the tech preview is that the resulting list of services from the initial kql at the time of group creation is static unless someone uses edit group -> refresh

Reply:
Thanks for the feedback! Addressed in [`51b860d`](https://github.com/elastic/kibana/pull/132070/commits/51b860d8daf6efd5ce68459e25271bb89be62079).